tests/unit_tests/ensemble_evaluator/ensemble_evaluator_utils.py

```diff
@@ -41,6 +41,15 @@ def send_dispatch_event(client, event_type, source, event_id, data, **extra_attr
     client.send(to_json(event1))
 
 
+async def send_dispatch_event_async(
```

Comment:
Do we still need the old synced `send_dispatch_event(...)`?

Reply:
Need to find out, but maybe not.

Reply:
Yes, it's needed for the `TestEnsemble` class, which sends "fake" events in the synced fashion. Once ee_async is in we can then remove it.

Reply:
Maybe we can remove it